backend/FacilityBooking/FacilitiesAPI.py

```diff
@@ -268,8 +268,15 @@ def add_booking():
             {"_id": 0}
         ).count()
```

Comment:
You only really need to remove the .count() here and change it to a list. This changes the return value of the db from the number of items that is matched to the items that is matched. 

Do some simple cleaning up (remove _id) and return this array